bluebottle/bb_accounts/views.py

```diff
@@ -59,7 +60,10 @@ def get_name(self):
         return "Users"
 
     def pre_save(self, obj):
-        obj.primary_language = self.request.LANGUAGE_CODE[:2]
+        if hasattr(properties, 'DEFAULT_SIGNUP_LANGUAGE'):
```

Comment:
Yeah makes sense, good point. Ill update the PR this weekend or maybe tonight.

Sent from my iPhone

> On 27 feb. 2015, at 09:10, rollick <notifications@github.com> wrote:
> 
> In bluebottle/bb_accounts/views.py:
> 
> > @@ -59,7 +60,10 @@ def get_name(self):
> >          return "Users"
> >  
> >      def pre_save(self, obj):
> > -        obj.primary_language = self.request.LANGUAGE_CODE[:2]
> > +        if hasattr(properties, 'DEFAULT_SIGNUP_LANGUAGE'):
> I don't think this is correct. We should check if obj.primary_language is in the list of supported LANGUAGES: if it is then set it for the user, otherwise use the LANGUAGE_CODE.
> 
> LANGUAGES and LANGUAGE_CODE are in the tenant properties.
> 
> —
> Reply to this email directly or view it on GitHub.
>